site/src/scaffold/standard.html

```diff
@@ -8,6 +8,7 @@
 		<meta property="og:title" content="{{title}}" />
 		<meta property="og:type" content="website" />
 		<meta name="color-scheme" content="light dark" />
+		<meta http-equiv="Content-Security-Policy" content="default-src 'none'; script-src 'self'; style-src 'self' 'unsafe-hashes' 'sha256-xcaKW0OyR0UBm0XPKdo2gWdgJW81zFPp1ovF7iW/x0c=' 'sha256-gyrAVcbmnI8a2UtMNEZq61n4Hd0+BbNaI7mhujFCsY0=' 'sha256-5lij/kYf1SCLu25NaZsAVsoxtnSyilNGmv0srOQE/NE='; connect-src 'self'; frame-src 'self' https://www.figma.com/">
```

Comment:
What's the storywith the unsafehashes, sha256 hashes? Can we simplify this this a default source of self and the frame source you've included here>

Reply:
There are some inline styles in the plugins/parcel-transformer-markdown-html/lib/index.js file that aren't allowed unless you allow unsafe-hashes and then give the hash of the css...Unfortunately I found out later its not well supported (safari and firefox don't have it at all). I think we will have to pull the inline style attributes out into a style tag and give it a nonce.

We can definitely simplify it by setting default source to self. We only used 'none' because we were trying to make it as restritive as possible.

We could update it to 
content="default-src 'self'; style-src 'self' nonce-######; frame-src 'self' https://www.figma.com/"

Reply:
Most of the inline styles were created before we had a good set of atomic classes to use. Perhaps they could be removed and replaced with something else. 

CC @ollips @v-mingho

Reply:
It's mainly the min/max-height: 30vh for the code blocks/atomic classes list. We've talked about trying to avoid making new classes just for that as a last resort.

Reply:
The code-block with filter causes issues, where we setting up the `min/max height` and `overflow-y` for the container. And there are no any generic classes for limiting the height at the moment in Atlas and I don't know if we should add it. That's why we've tried a different way around with hashes yesterday. I also tested locally different approach - with moving styles into the `<style>`  and assigning `nonce` to it and seems to be working as well. Let's discuss offline what we want to do

Reply:
there is also some inline styles in the button.md file for color/background-color

Reply:
Yes, which btw should be updated with hex values instead of words, cause it's not working on a page as it should.